packages/backend-common/src/service/createServiceBuilder.ts

```diff
@@ -19,8 +19,8 @@ import { ServiceBuilder } from './types';
 
 /**
  * Creates a new service builder.
- *
  * @public
+ * @deprecated We are going to deprecated this old way of creating services in a near future, if you are using this service helper, please checkout the migration guide and make sure you migrate your backend to use the new system: https://backstage.io/docs/backend-system/building-backends/migrating.
```

Comment:
The guide that really should contain this content I'd say is https://backstage.io/docs/backend-system/building-plugins-and-modules/migrating, given that this is for plugins

Reply:
Update, thanks 🙏🏻